lib/concurrent/executor/java_thread_pool_executor.rb

```diff
@@ -74,7 +75,7 @@ def initialize(opts = {})
         @executor = java.util.concurrent.ThreadPoolExecutor.new(
```

Comment:
Not sure how to deal with the assignment `@executor =`  here. This is ok on JRuby (depending on undocumented volatile ivar access) but not ok in general. In general it should be synchronized or fixed by `ensure_ivar_visibility!`